tests/test_fx/test_ckpt_solvers/test_ckpt_torchvision.py

```diff
@@ -62,13 +62,13 @@ def check_backward_consistency(m: torch.nn.Module, gm: GraphModule, solver: Call
 
 def _run_ckpt_solver(rank):
     colossalai.launch(config={}, rank=rank, world_size=1, host='localhost', port=free_port(), backend='nccl')
-    MODEL_LIST = [tm.resnet18, tm.densenet121]
+    MODEL_LIST = [tm.densenet121]
```

Comment:
Why removed resnet?

Reply:
remove resnet18 because solver needs different memory budget to guarantee an activation checkpoint on it, so it might make the code awkward.

Reply:
You might build a dictionary for testing.
```python
dict = {
    tm.resnet18: (budget1, budget2, ...),
    tm.densenet121: (budget1, budget2, ...),
}
```
But anyway, it's not so important.

Reply:
Yes, I think it is not necessary, I think next time when this rotor solver is stable, we could remove greedy solver as it is in fact doesn't fit our demand (I think no one wants to use this as it doesn't aim to make good use of memory budget), at that time we could refactor this test.